protos/flyteidl/plugins/raw_container.proto

```diff
@@ -0,0 +1,34 @@
+syntax = "proto3";
+
+package flyteidl.plugins;
+
+option go_package = "github.com/lyft/flyteidl/gen/pb-go/flyteidl/plugins";
+
+// This configuration allows executing raw containers in Flyte using the Flyte CoPilot system.
+// Flyte CoPilot, eliminates the needs of flytekit or sdk inside the container. Any inputs required by the users container are side-loaded in the input_path
+// Any outputs generated by the user container - within output_path are automatically uploaded.
+// We are starting this as a plugin, but after the container within Pod ordering stabilizes, we can move this
+// into the default task definition.
+message CoPilot {
+  // MetadataFormat is used to decide the format in which the input metadata so that the users containers can read it. If the user has access to the protocol buffer definitions, 
+  // it is recommended to use the PROTO format.
+  // JSON and YAML do not need any protobuf definitions to read it
+  enum MetadataFormat {
+    JSON = 0;
```

Comment:
why is JSON the defaultr?

Reply:
because, it is assumed that users of raw container plugin do not have flytekit and hence cannot easily read protobufs